src/OAuth2.php

```diff
@@ -83,7 +87,12 @@ public function buildAuthUrl(array $params = [])
         if ($this->validateAuthState) {
             $authState = $this->generateAuthState();
             $this->setState('authState', $authState);
-            $defaultParams['state'] = $authState;
+            $this->addToOauthState('authState', $authState);
+        }
+
+        $state = $this->getOauthState();
+        if ($state) {
+            $defaultParams['state'] = $state;
```

Comment:
I think now we put to `$defaultParams['state']` json string (`getOauthState` use json_encode()), but earlier we put just a string from https://github.com/xchwarze/yii2-authclient/blob/67fd4e5db42dd285af7d4c128e25b2fe1affb825/src/OAuth2.php#L210-L222